src/skin/firstRun.html

```diff
@@ -28,6 +28,18 @@ <h1 class="i18n_name"></h1>
       </div>
     </header>
 
+    <!-- Overlay nudging users to pin their extension (only shown on chromium browsers) -->
+    <div id="overlay" style="display:none;" class="nudge-overlay"></div>
+    <div id="pin-nudge" style="display:none;" class="nudge-overlay">
```

Comment:
Why is the nudge-overlay class for?

Reply:
just something I forgot to remove! thanks for catching